Packs/Core/Playbooks/playbook-SSO_Brute_Force.yml

```diff
@@ -0,0 +1,1051 @@
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: f0f71c4c-c584-4c50-802f-abc9477770d1
+    type: regular
+    task:
+      id: f0f71c4c-c584-4c50-802f-abc9477770d1
+      version: -1
+      name: Get login event attempt information
+      description: Returns information about each alert ID.
+      script: '|||core-get-cloud-original-alerts'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      alert_ids:
+        complex:
+          root: alert
+          accessor: id
+          transformers:
+          - operator: uniq
+      filter_alert_fields:
+        simple: "false"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1320,
+          "y": 150
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "14":
+    id: "14"
+    taskid: 1133c2aa-8085-44e8-8708-791dc0ea0a95
+    type: regular
+    task:
+      id: 1133c2aa-8085-44e8-8708-791dc0ea0a95
+      version: -1
+      name: Check if the user risk level is high
+      description: Retrieve the risk score of a specific user or list of users with the highest risk score in the environment along with the reason affecting each score.
+      script: '|||core-list-risky-users'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "16"
+    scriptarguments:
+      user_id:
+        simple: ${alert.username.[0]}
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1330,
+          "y": 620
+        }
+      }
+    note: true
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: 1da4e82d-8915-4d26-8226-2a03b9464bd3
+    type: title
+    task:
+      id: 1da4e82d-8915-4d26-8226-2a03b9464bd3
+      version: -1
+      name: Containment
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "21"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1090,
+          "y": 800
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: 3b710eeb-d4b7-428a-8eb7-f30a4e06e716
+    type: condition
+    task:
+      id: 3b710eeb-d4b7-428a-8eb7-f30a4e06e716
+      version: -1
+      name: Any suspicious evidences identified during the investigation phase?
```

Comment:
change to "Checking if suspicious evidence where found during the investigation phase"